Baekjoon/l-lyun/11047.py

```diff
@@ -0,0 +1,27 @@
+'''
+작성자: l-lyun
+문제 링크: https://www.acmicpc.net/problem/11047
+카테고리: 그리디
+
+k == coin일 때 누락
+현재 조건문 k >= coin에서, 기존 답안은 등호 명시 안해줘서 오답 발생
+'''
+import sys
+
+n, k = map(int, sys.stdin.readline().split())
```

Comment:
```suggestion
input = sys.stdin.readline
n, k = map(int, input().split())
```

`input = sys.stdin.readline`으로 input에 함수를 할당하는 방법이 cs문제 풀이에서 일반적으로 사용되는 것 같기도해요.